### PR TITLE
Error monitoring

### PR DESCRIPTION
**Pull request by hanars**
We are going to add alerting whenever any errors are logged, so before I do that I wanted to clean up what is logged as an error and what is logged as a warning

### CODE REVIEW

**Comment by hanars** [comment]
passing tests: https://travis-ci.org/github/broadinstitute/seqr/builds/742550218

**Comment by hanars** [comment]
codacy: https://app.codacy.com/gh/macarthur-lab/seqr/pullRequest?prid=6479835

File: seqr/utils/elasticsearch/es_search.py [hunk 1]
```diff
@@ -1037,7 +1037,7 @@ def _liftover_grch38_to_grch37():
         try:
             LIFTOVER_GRCH38_TO_GRCH37 = LiftOver('hg38', 'hg19')
         except Exception as e:
-            logger.warn('WARNING: Unable to set up liftover. {}'.format(e))
+            logger.error('WARNING: Unable to set up liftover. {}'.format(e))
```
Comment (ShifaSZ) [inline seqr/utils/elasticsearch/es_search.py:1040]: Using `logger.error()` but there is `WARNING` in the message. Should the `WARNING` be changed to `ERROR`?

File: seqr/utils/elasticsearch/es_search.py [hunk 2]
```diff
@@ -1048,7 +1048,7 @@ def _liftover_grch37_to_grch38():
         try:
             LIFTOVER_GRCH37_TO_GRCH38 = LiftOver('hg19', 'hg38')
         except Exception as e:
-            logger.warn('WARNING: Unable to set up liftover. {}'.format(e))
+            logger.error('WARNING: Unable to set up liftover. {}'.format(e))
```
Comment (ShifaSZ) [inline seqr/utils/elasticsearch/es_search.py:1051]: As above.  Should the WARNING be changed to ERROR?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]
There are two questions.

File: seqr/utils/elasticsearch/es_search.py (hunk 2 above)
Comment (hanars) [reply seqr/utils/elasticsearch/es_search.py]: done

File: seqr/utils/elasticsearch/es_search.py (hunk 1 above)
Comment (hanars) [reply seqr/utils/elasticsearch/es_search.py]: done

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**